src/formats/MMTF.cpp

```diff
@@ -392,6 +526,24 @@ void MMTFFormat::write(const Frame& frame) {
 }
 
 void MMTFFormat::add_residue_to_structure(const Frame& frame, const Residue& residue) {
+
+    if (frame.cell().shape() != UnitCell::INFINITE) {
```

Comment:
I though I already left this comment, but I cant' find it now 🤷‍♂️ .

Instead of checking for infinite cell, we should check that the same cell is used for all frames, i.e. refuse `cell(10, 12, 10)` and then `cell(10, 10, 10)`.

Reply:
I have an email with a similar comment, but it never appeared here! Should be addressed now.